content/en/docs/installing-on-kubernetes/_index.md

```diff
@@ -63,4 +63,14 @@ Remove KubeSphere from Kubernetes clusters.
 
 Below you will find some of the most viewed and helpful pages in this chapter. It is highly recommended that you refer to them first.
 
```

Comment:
Do we need to move the **"Most Popular Pages"** section to the beginning? This might be easier for new users to find them.

![image](https://user-images.githubusercontent.com/40452856/94112275-c6514680-fe77-11ea-9402-a622787354ac.png)